sway-lib-std/src/option.sw

```diff
@@ -51,10 +49,11 @@ impl<T> Option<T> {
     /// Instead, prefer to use pattern matching and handle the [`None`]
     /// case explicitly.
     fn unwrap(self) -> T {
-        if let Option::Some(inner_value) = self {
-            inner_value
-        } else {
-            revert(0);
+        match self {
+            Option::Some(inner_value) => inner_value,
+            _ => {
```

Comment:
Is there a need to use `{  }` here? Does `revert` not half control flow analysis properly? If not, should an issue be filed, or is that expected behavior?

Reply:
There is no reason to use `{ }`, this was just my personal code style habit. I updated it to match the style of the other branch.